src/main/kotlin/uk/gov/dluhc/printapi/config/MessagingConfiguration.kt

```diff
@@ -37,40 +35,50 @@ class MessagingConfiguration {
+    ) = MessagingConfigurationHelper.sqsTemplate(sqsAsyncClient, sqsMessagingMessageConverter)
 
     @Bean
-    fun processPrintResponseFileQueue(queueMessagingTemplate: QueueMessagingTemplate) =
-        MessageQueue<ProcessPrintResponseFileMessage>(processPrintResponseFileQueueName, queueMessagingTemplate)
+    fun processPrintRequestBatchQueue(sqsTemplate: SqsTemplate) =
+        MessageQueue<ProcessPrintRequestBatchMessage>(processPrintRequestBatchQueueName, sqsTemplate)
 
     @Bean
-    fun processPrintResponseQueue(queueMessagingTemplate: QueueMessagingTemplate) =
-        MessageQueue<ProcessPrintResponseMessage>(processPrintResponseQueueName, queueMessagingTemplate)
+    fun processPrintResponseFileQueue(sqsTemplate: SqsTemplate) =
+        MessageQueue<ProcessPrintResponseFileMessage>(processPrintResponseFileQueueName, sqsTemplate)
 
     @Bean
-    fun removeCertificateQueue(queueMessagingTemplate: QueueMessagingTemplate) =
-        MessageQueue<RemoveCertificateMessage>(removeCertificateQueueName, queueMessagingTemplate)
+    fun processPrintResponseQueue(sqsTemplate: SqsTemplate) =
+        MessageQueue<ProcessPrintResponseMessage>(processPrintResponseQueueName, sqsTemplate)
 
     @Bean
-    fun triggerVoterCardStatisticsUpdateQueue(queueMessagingTemplate: QueueMessagingTemplate) =
-        MessageQueue<UpdateStatisticsMessage>(triggerVoterCardStatisticsUpdateQueueName, queueMessagingTemplate)
+    fun removeCertificateQueue(sqsTemplate: SqsTemplate) =
+        MessageQueue<RemoveCertificateMessage>(removeCertificateQueueName, sqsTemplate)
 
     @Bean
-    fun queueMessageHandlerFactory(
-        jacksonMessageConverter: MappingJackson2MessageConverter,
-        hibernateValidator: Validator
-    ): QueueMessageHandlerFactory =
-        QueueMessageHandlerFactory().apply {
-            setArgumentResolvers(
-                listOf(
-                    HeadersMethodArgumentResolver(),
-                    NotificationSubjectArgumentResolver(),
-                    AcknowledgmentHandlerMethodArgumentResolver("Acknowledgment"),
-                    VisibilityHandlerMethodArgumentResolver("Visibility"),
-                    PayloadMethodArgumentResolver(jacksonMessageConverter, hibernateValidator)
-                )
-            )
-        }
+    fun triggerVoterCardStatisticsUpdateQueue(sqsTemplate: SqsTemplate) =
+        MessageQueue<UpdateStatisticsMessage>(triggerVoterCardStatisticsUpdateQueueName, sqsTemplate)
+
+    @Bean
+    fun sqsMessagingMessageConverter(
+        objectMapper: ObjectMapper
+    ) = MessagingConfigurationHelper.sqsMessagingMessageConverter(objectMapper)
+
+    @Bean
+    fun defaultSqsListenerContainerFactory(
+        sqsAsyncClient: SqsAsyncClient,
+        sqsMessagingMessageConverter: SqsMessagingMessageConverter,
+    ) = MessagingConfigurationHelper.defaultSqsListenerContainerFactory(
+        sqsAsyncClient,
+        sqsMessagingMessageConverter,
+        5, // TODO EIP1-9472: Configurable?
```

Comment:
Outstanding TODO here - do we want this to be configurable? At least extracted out to a named constant would be good to make it clearer what the value is.